back-end/resources/testing/testComponentConstraintNotPresent.json

```diff
@@ -7,14 +7,100 @@
   },
   "devices": [
     {
-      "id": "ledLights",
+      "id": "controlBoard",
+      "description": "Control board with three switches, three slides with lights and one main switch.",
+      "input": {
+        "redSwitch": "boolean",
+        "orangeSwitch": "boolean",
+        "greenSwitch": "boolean",
+        "slider1": "numeric",
+        "slider2": "numeric",
+        "slider3": "numeric",
+        "mainSwitch": "boolean"
+      },
+      "output": {
+        "greenLight1": {
+          "type": "string",
+          "instructions": {
+            "interval": "numeric",
```

Comment:
change instruction here as well, and also all other files.

Reply:
These are test file, this is unnecessary work